src/main/java/org/graphqlcrudjava/types/TypeMapImpl.java

```diff
@@ -1,48 +1,51 @@
 package org.graphqlcrudjava.types;
 
+import graphql.Scalars;
+import graphql.schema.GraphQLOutputType;
+
 import java.sql.Types;
 
 public class TypeMapImpl implements TypeMap {
 
     @Override
-    public String getAsGraphQLTypeString(int dataType) {
-        String typeString;
+    public GraphQLOutputType getAsGraphQLTypeString(int dataType) {
+        GraphQLOutputType typeString;
         switch (dataType) {
             case Types.TINYINT:
             case Types.INTEGER:
             case Types.SMALLINT:
-                typeString = "Int";
+                typeString = Scalars.GraphQLInt;
                 break;
             case Types.CHAR:
             case Types.VARCHAR:
             case Types.LONGVARCHAR:
-                typeString = "String";
+                typeString = Scalars.GraphQLString;
                 break;
             case Types.DOUBLE:
             case Types.FLOAT:
             case Types.REAL:
             case Types.NUMERIC:
             case Types.DECIMAL:
-                typeString = "Float";
-                break;
-            case Types.DATE:
-            case Types.TIMESTAMP:
-            case Types.TIME:
-                typeString = "Date";
+                typeString = Scalars.GraphQLFloat;
                 break;
+//            case Types.DATE:
+//            case Types.TIMESTAMP:
+//            case Types.TIME:
+//                typeString = "Date";
+//                break;
             case Types.BIT:
-                typeString = "Boolean";
-                break;
-            case Types.OTHER:
-                typeString = "Object";
-                break;
-            case Types.BINARY:
-            case Types.VARBINARY:
-            case Types.LONGVARBINARY:
-                typeString = "Object";
-                break;
+                typeString = Scalars.GraphQLBoolean;
+                break;
+//            case Types.OTHER:
```

Comment:
Requires logging issue

Reply:
> Yep. Feel free to work on this PR to add extra stuff. If you prefer us to merge please do it. No worries from me. Codebase is small so no worries. I would add more TODOS into code - there are some commented parts and we will quickly forgot why we commented them :D
> Changes are small so this can land to master no problem

yes, we can proceed that way. 👍